homeassistant/components/mqtt/mixins.py

```diff
@@ -1006,6 +1011,7 @@ class MqttEntity(
     """Representation of an MQTT entity."""
 
     _attr_should_poll = False
+    _default_name: str | None | UndefinedType
```

Comment:
Why is `UndefinedType` allowed here, it doesn't seem to be used?

Reply:
See: https://github.com/home-assistant/core/pull/95159#discussion_r1245431685
But you are right. It isn't used. I can remove it.